<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Runs::RunFlagsDialog</class>
 <widget class="QDialog" name="Runs::RunFlagsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run flags dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbxDisqualified">
     <property name="text">
      <string>Disqualified</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbxMisPunch">
     <property name="text">
      <string>Missed punch</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbxBadCheck">
     <property name="toolTip">
      <string>Check time on SI card is to early</string>
     </property>
     <property name="text">
      <string>Bad check</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbxNotStart">
     <property name="text">
      <string>Did not start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbxNotFinish">
     <property name="text">
      <string>Did not finish</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbxNotCompeting">
     <property name="text">
      <string>Not competing</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbxDisqualified</tabstop>
  <tabstop>cbxMisPunch</tabstop>
  <tabstop>cbxBadCheck</tabstop>
  <tabstop>cbxNotStart</tabstop>
  <tabstop>cbxNotFinish</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Runs::RunFlagsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Runs::RunFlagsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
